torch/lib/c10d/ProcessGroupNCCL.cpp

```diff
@@ -212,6 +229,10 @@ void ProcessGroupNCCL::WorkNCCL::synchronize() {
         // can not run new events successfully.
         for (const auto& ncclComm : ncclComms_) {
           ncclComm->ncclCommAbort();
```

Comment:
here we abort ncclComm as well, but we did not add it to abortedComms_() because this will trigger nccl error and will be caught by watchdog, then watch dog will add this id to abortedComms_, right?